### ISSUE TITLE
A few challenges

### ISSUE THREAD

**Issue by bc-0dp** [bigcommerce/bigcommerce-api-node#24]
Package name seems to be `@bigcommerce/api-nodejs` not `bigcommerce-api-node`

When I try and instantiate a new `RestClient`
```
const bc = new BigCommerce.Rest({
    accessToken: process.env.ACCESS_TOKEN,
    storeHash: process.env.STORE_HASH
});
```
I get

 `TypeError: BigCommerce.RestClient is not a constructor`

**Comment by bc-0dp** [comment]
It seems that ES imports are not supported as advertised, instead its a common.js module
`const BigCommerce = require("@bigcommerce/api-nodejs").default;`

Creating a wrapper can 'solve' this
```wrapper.cjs
const BigCommerce = require("@bigcommerce/api-nodejs").default;

module.exports = BigCommerce;
```

that can then be imported elsewhere `import BigCommerce from './wrapper.cjs';`

**Comment by matthewvolk** [comment]
@bc-chaz @bc-nekisham Note, consumers of this package won't be able to make use of this new syntax until the package is published with the latest changes from #30

@bc-0dp Once the new version is published, it would be great if you could update the version of this package in your `package.json` and test this out to see if the issue is resolved on your end 👍
